frontend-react/package.json

```diff
@@ -182,13 +184,21 @@
     "terser-webpack-plugin": "^5.3.6",
     "ts-node": "^10.9.1",
     "tslib": "^2.5.0",
-    "typescript": "^4.9.5",
+    "typescript": "^5.0.2",
     "webpack": "^5.76.0",
     "webpack-bundle-analyzer": "^4.7.0",
     "webpack-dev-server": "^4.11.1",
     "webpack-manifest-plugin": "^5.0.0",
     "workbox-webpack-plugin": "^6.5.4"
   },
+  "resolutions": {
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
```

Comment:
Oh, interesting!  What else uses this dependency?

Reply:
react-dev-utils, a sub-package of CRA that is targeted towards a 6.X.X version of fork-ts-checker-webpack-plugin. we will need to do more work to drop all CRA attachments